Sources/Private/Model/Keyframes/KeyframeGroup.swift

```diff
@@ -7,6 +7,8 @@
 
 import Foundation
 
+// MARK: - KeyframeGroup
+
 /**
  Used for coding/decoding a group of Keyframes by type.
```

Comment:
Should this note be reworded now that this is part of the public API? And should this be moved out of the private folder?

Reply:
I kept `KeyframeGroup` internal in favor of just using `[Keyframe]` in the public API. (`KeyframeGroup` is basically just a thin wrapper around an array that implements `Codable`, which we don't actually need in the public API).